packages/@react-types/combobox/src/index.d.ts

```diff
@@ -34,16 +34,19 @@ export interface ComboBoxProps<T> extends CollectionBase<T>, SingleSelection, In
   //  * @default 'suggest'
   //  */
   // completionMode?: 'suggest' | 'complete',
-  /**
-   * The interaction required to display the ComboBox menu.
-   * @default 'input'
-   */
-  menuTrigger?: MenuTriggerAction,
+ /**
+  * The interaction required to display the ComboBox menu.
+  * @default 'input'
+  */
+  menuTrigger?: MenuTriggerAction
+}
+
+export interface AriaComboBoxProps<T> extends ComboBoxProps<T>, DOMProps {
```

Comment:
Now that you have `DOMProps` here, can get rid of it from `ComboBoxProps` since stately doesn't use those